Lib/test/test_uu.py

```diff
@@ -218,6 +220,20 @@ def test_decodetwice(self):
         with open(self.tmpin, 'rb') as f:
             self.assertRaises(uu.Error, uu.decode, f)
 
+    def test_decode_mode(self):
+        # Verify that decode() will set the given mode for the out_file
+        expected_mode = 0o444
+        with open(self.tmpin, 'wb') as f:
+            f.write(encodedtextwrapped(expected_mode, self.tmpout))
+
+        with open(self.tmpin, 'rb') as f:
+            uu.decode(f)
+
+        self.assertEqual(stat.S_IMODE(
+            os.stat(self.tmpout).st_mode), expected_mode)
+        # make file writable again, so it can be removed (Windows only)
+        self.addCleanup(os.chmod, self.tmpout, expected_mode | stat.S_IWRITE)
+
 
-if __name__=="__main__":
+if __name__ == "__main__":
```

Comment:
I know this may be done by your editor/IDE automatically, but please don't include cosmetic changes to the PR.

Reply:
We tend to reject cosmetic-only PRs, but if you find similar cases in `Lib/test/` (not in the standard library) and have time to prepare a PR, it may be accepted.